rest/client/methods.go

```diff
@@ -188,9 +187,7 @@ func (c *communicatorImpl) GetProjectRef(ctx context.Context, taskData TaskData)
 	return projectRef, nil
 }
 
-// GetDistro returns the distro for the task.
-func (c *communicatorImpl) GetDistro(ctx context.Context, taskData TaskData) (*distro.Distro, error) {
-	d := &distro.Distro{}
+func (c *communicatorImpl) GetDistroView(ctx context.Context, taskData TaskData) (*apimodels.DistroView, error) {
 	info := requestInfo{
 		method:   get,
 		taskData: &taskData,
```

Comment:
This should be updated to have the distro_view path suffix shouldn't it?

Reply:
Ah, good catch. Fixed it.